Marlin/configuration_store.cpp

```diff
@@ -64,28 +64,31 @@
  *  195  M206 XYZ  home_offset                      (float x3)
  *  207  M218 XYZ  hotend_offset                    (float x3 per additional hotend)
  *
- * Global Leveling:
+ * Global Leveling:                                 6 bytes
  *  219            z_fade_height                    (float)
+ *  223            GRID_MAX_POINTS_X                (uint8_t)
+ *  224            GRID_MAX_POINTS_Y                (uint8_t)
```

Comment:
> thinkyhead made some cosmetic changes prior to merging this PR. Not sure why they were necessary prior to merging, 

These additions help keep things well documented.     And in a perfect world, we don't want a bunch of commits that comprise 'one change'.   That makes it very difficult to back stuff out or understand a change.    

@thinkyhead    I didn't merge this even though it was UBL because it caused changes to the EEPROM.    If you are OK with the changes...  It would be helpful to us to merge it.